stubs/openssl-python/OpenSSL/crypto.pyi

```diff
@@ -1,197 +1,210 @@
-import sys
 from datetime import datetime
-from typing import Callable, Iterable, List, Optional, Set, Text, Tuple, Union
+from typing import Callable, Iterable, List, Optional, Sequence, Set, Text, Tuple
 
-from cryptography.hazmat.primitives.asymmetric import dsa, rsa
+from cryptography.hazmat.primitives.asymmetric.dsa import DSAPrivateKey, DSAPublicKey
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey, RSAPublicKey
+from cryptography.x509 import Certificate, CertificateRevocationList, CertificateSigningRequest
+
+_Key = DSAPrivateKey | DSAPublicKey | RSAPrivateKey | RSAPublicKey
```

Comment:
Unfortunately, the new syntax doesn't work in type aliases, because type checkers ...

```suggestion
_Key = Union[DSAPrivateKey, DSAPublicKey, RSAPrivateKey, RSAPublicKey]
```

Reply:
Fixed.